### PR TITLE
Tensorflow and Keras

### PR DESCRIPTION
**Pull request by abhiraghuram**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 # DSC Devsprints Jan 2020
 ## Date and Venue - TBD
+#Project Suggestion - Here are some of the links of the projects I found interesting
```
Comment (rohitsanj) [inline README.md]: Remove this line

File: README.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
 Has contributing to Open source projects been one of those plans for you that just remained a plan? Or did you try contributing to an Open source project but got lost somewhere in the middle due to lack of guidance? Well, Devsprints are here to save the day! During Devsprints, Mentors/Contributors of Open source projects guide participants through everything, from generating the development build to submitting a patch for a bug or improve the project's documentation.
 
 We will also be conducting a brief workshop on Git.
 Links we will refer to:
 https://www.digitalocean.com/community/tutorials/how-to-use-git-a-reference-guide
-
+
```
Comment (rohitsanj) [inline README.md:14]: Remove this extra line

File: README.md [hunk 3]
```diff
@@ -26,3 +27,5 @@ TBD
 |lenskit(rec system)  |https://lenskit.org/  | https://github.com/lenskit/lkpy
 |Submitty  | https://submitty.org/       | https://submitty.org/developer/how_to_contribute
 |NumPy  |https://numpy.org/     | https://docs.scipy.org/doc/numpy/dev/
+|TensorFlow         | https://www.tensorflow.org           | https://www.tensorflow.org/tutorials
+|Keras              |https://keras.io                      |
```
Comment (rohitsanj) [inline README.md]: Add the contributing guide for tf and keras. not tutorials link.

**Review by rohitsanj — COMMENTED** [review]
Pl make these changes

**Review by abhiraghuram — COMMENTED** [review]
Made changes

**Pull request merged**